cubedash/templates/arrivals.html

```diff
@@ -7,6 +7,10 @@
     <div class="panel highlight">
         <h2 class="followed lonesome">Recent Dataset Arrivals</h2>
         <span class="header-follow">over {{ period_length.days }} days</span>
+    <a href="{{ url_for('stac.arrivals') }}" class="badge header-badge">
+        As Stac Catalog
+        <i class="fa fa-file-text-o" aria-hidden="true"></i>
+    </a>
```

Comment:
indent more on this `<a>` tag?

Reply:
leaving this for later 👍 have questions around static html/js minification for explorer we can revisit this with. Also possible improvements with CDN.